Project-I-Social-Card/social-card/src/App.js

```diff
@@ -1,23 +1,35 @@
 import React from 'react';
 import './App.css';
+import ImageThumb from './components/HeaderComponents/ImageThumbnail';
+import HeaderContent from './components/HeaderComponents/HeaderContent';
+import CardBanner from './components/CardComponents/CardBanner';
+import CardContent from './components/CardComponents/CardContent';
+import CardContainer from './components/CardComponents/CardContainer';
+import HeaderTitle from './components/HeaderComponents/HeaderTitle';
+import Footer from './components/FooterComponents/Footer';
+
 
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Social Card!</h3>
-      <p>
-        Begin by exploring the `components` directory. You'll notice we have a
-        few files that we've already included in there to get you started right
-        away building components. You'll need to make sure you include your
-        components that you build in this file to watch your app come to life
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+
+    <div className='Container'>
+
+      <div className='Header'>
+        <ImageThumb />
+      <div>
+        <HeaderTitle />
+        <HeaderContent />
+      </div>
+
     </div>
+
+      <div className='Card'>
+        <CardContainer />
+        <CardBanner />
+        <CardContent />
+      </div>
+            <Footer />
+      </div>
```

Comment:
This is another way of structuring your App.js
```suggestion
<div className="container">
  <HeaderContainer />
  <CardContainer />
  <Footer/>
</div>
```
Then, inside each of these components, you would nest other components based on their responsibility.